### PR TITLE
Update Razor output for version 6

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to https://github.com/aspnet/Announcements/issues/459.

### CODE REVIEW

File: aspnetcore/razor-pages/ui-class.md [hunk 1]
```diff
@@ -319,7 +319,7 @@ The *_ViewStart.cshtml* file is required to use the layout of the Razor Pages pr
   dotnet build RazorUIClassLib
   ```
 
-The build output contains *RazorUIClassLib.dll* and *RazorUIClassLib.Views.dll*. *RazorUIClassLib.Views.dll* contains the compiled Razor content.
+In ASP.NET Core 6 and later versions, the build output contains *RazorUIClassLib.dll*. In earlier versions, the build output contains two assemblies: *RazorUIClassLib.dll* and *RazorUIClassLib.Views.dll*, which contains the compiled Razor content.
```
Comment (guardrex) [inline aspnetcore/razor-pages/ui-class.md:322]: Could version it (feel free to edit this comment directly) ...

```suggestion
::: moniker range=">= aspnetcore-6.0"

The build output contains `RazorUIClassLib.dll`.

::: moniker-end

::: moniker range="< aspnetcore-6.0"

The build output contains `RazorUIClassLib.dll` and `RazorUIClassLib.Views.dll`. `RazorUIClassLib.Views.dll` contains the compiled Razor content.

::: moniker-end
```

**Comment by gewarren** [comment]
image

There's actually a build warning here. Would you like to try out my status checker GitHub Action that would have flagged this as a problem? If so, I can create a PR to add the workflow.

**Comment by guardrex** [comment]
My bad ... it's nested ... and they can't be nested. It looks like your original approach would have been better.

**Comment by guardrex** [comment]
I'll fix it. Stand-by.

**Comment by Rick-Anderson** [comment]
> My bad ... it's nested ... and they can't be nested. It looks like your original approach would have been better.

Plus nested would get too complicated.

File: aspnetcore/mvc/views/view-compilation.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ A Razor file is compiled at runtime, when the associated Razor Page or MVC view
 
 ## Razor compilation
 
-Build- and publish-time compilation of Razor files is enabled by default by the Razor SDK. Editing Razor files after they're updated is supported at build time. By default, only the compiled *Views.dll* and no *.cshtml* files or references assemblies required to compile Razor files are deployed with your app.
+Build-time and publish-time compilation of Razor files is enabled by default by the Razor SDK. Editing Razor files after they're updated is supported at build time. By default, only the compiled *Views.dll* and no *.cshtml* files or references assemblies required to compile Razor files are deployed with your app.
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/views/view-compilation.md:163]:
```suggestion
Build-time and publish-time compilation of Razor files is enabled by default by the Razor SDK. Editing Razor files after they're updated is supported at build time. By default, only the compiled *Views.dll* and no *.cshtml* files or references assemblies required to compile Razor files are deployed with an app.
```

**Comment by guardrex** [comment]
I didn't see it and didn't check. 🙈 

I'll fix it up. Stand-by.

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/mvc/views/view-compilation.md (hunk 2 above)
Comment (guardrex) [reply aspnetcore/mvc/views/view-compilation.md:163]: Hold off on merging this suggestion ... I'll do it manually on my commit.

Comment (guardrex) [reply aspnetcore/mvc/views/view-compilation.md:163]: Oooops! Other file ...... so much for cooking dinner 👨‍🍳 and working on the repo!

**Comment by guardrex** [comment]
btw @Rick-Anderson ... Line 322 is in the block of content that's versioned <3.0. Therefore, any remarks there for 6.0 or later won't be seen. There is no matching remark in the >=3.0 content AFAICT.

I'll let you take over here now. **_I've done enough damage here!_** 😄😢😄😢😄😢🙈

File: aspnetcore/mvc/views/view-compilation.md (hunk 2 above)
Comment (Rick-Anderson) [reply aspnetcore/mvc/views/view-compilation.md]: Did you mean  🍻  with dinner 🤣

File: aspnetcore/razor-pages/ui-class.md [hunk 3]
```diff
@@ -319,7 +319,7 @@ The *_ViewStart.cshtml* file is required to use the layout of the Razor Pages pr
   dotnet build RazorUIClassLib
   ```
 
-The build output contains *RazorUIClassLib.dll* and *RazorUIClassLib.Views.dll*. *RazorUIClassLib.Views.dll* contains the compiled Razor content.
+In ASP.NET Core version 6.0 or later, the build output contains `RazorUIClassLib.dll`. In versions of ASP.NET Core earlier than 6.0, the build output contains `RazorUIClassLib.dll` and `RazorUIClassLib.Views.dll`. `RazorUIClassLib.Views.dll` contains the compiled Razor content.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/ui-class.md:322]: Can you revert this and I'll make those changes in #22013

In this repo, we never say in version x and later ...
In previous version ....

We use monikers. That was a decision the PU made with V 1.0

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@gewarren  feel free to assign ASP.NET Core 6.0 changes to me. But any help is always welcome.

**Pull request merged**